mass_mailing_contact_partner/models/res_partner.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    mailing_contact_id = fields.Many2one(
+        comodel_name="mailing.contact",
+        string="Mass Mailing Contact",
+        help="The mailing list contact that matches this partner's email.",
+        compute="_compute_mailing_contact_id",
+        store=True,
+    )
+
+    @api.depends("email_normalized")
+    def _compute_mailing_contact_id(self):
+        # Unique list of normalized email addresses
+        emails = {m for m in self.mapped("email_normalized") if m}
+        if not emails:
+            self.mailing_contact_id = False
```

Comment:
```suggestion
            self.update({"mailing_contact_id": False})
```

Reply:
Is there a reason for this? Same result but less readable IMO 🤔

Reply:
nvm :)